SAPHana/bin/SAPHanaSR-manageAttr

```diff
@@ -38,7 +39,7 @@ cmd=$(basename "$exe")
 ##### functions
 # USAGE
 usage() {
-    echo "USAGE:    $0 --sid=<SID> [--ino=<INSTNO>] --case <use case> --migrate <resource> | --check"
+    echo "USAGE:    $0 --sid=<SID> --ino=<INSTNO> --case <use case> --migrate <resource> | --check"
```

Comment:
Is the ino now mandatory? Why? For eveluating the attributes we only need the SID... Maybe I oversee something :)

Reply:
If I need the setting of  'OCF_RESKEY_InstanceNumber' when calling 'SAPHanaController status', then the 'ino' needs to be mandatory. If it's sufficient to only use 'OCF_ROOT' and ' OCF_RESKEY_SID' for this call, then we do not need the 'ino'

Reply:
I have removed the OCF_RESKEY_InstanceNumber' parameter from the 'SAPHanaController status' call, so the 'ino' is not mandatory.
But if we need the instance number additional to the SID to get reliable the msl- and cln- resources, then we need to make it mandatory